docs/design/authentication-patterns.md

```diff
@@ -0,0 +1,68 @@
+# Authentication
+
+Use  dialogs to present authentication screens to your end users. Our flows provide a seamless experience that integrates the Microsoft sign-in with your brand.
```

Comment:
Add justification for authentication fx enable your end users to sign into your service or Microsoft Graph.